### PR TITLE
Update DE translation

### PR DESCRIPTION
**Pull request by agua8**

### CODE REVIEW

File: Translations/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -233,21 +233,21 @@
...
-"Gift Keka a couple coffees each month" = "Schenke Keka monatlich ein Paar Kaffee";
-"Monthly breakfast" = "Monatliches Frühstück";
+"Monthly pair of coffees" = "Monatlich ein paar Kaffees";
+"Gift Keka a couple coffees each month" = "Schenke Keka jeden Monat ein paar Kaffees";
+"Monthly breakfast" = "Monatlich ein Frühstück";
 "Gift Keka a breakfast each month" = "Schenke Keka jeden Monat ein Frühstück";
-"Tip a coffee" = "Gebe Trinkgeld für Kaffee";
-"Tip Keka a coffee so it can stay awake" = "Spende einen Kaffee, damit Keka wach bleib";
+"Tip a coffee" = "Spende einen Kaffee";
+"Tip Keka a coffee so it can stay awake" = "Spende Keka einen Kaffee, damit es wach bleibt";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:245]: Must be 45 characters or less

File: Translations/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -233,21 +233,21 @@
...
 "Tip a breakfast" = "Spende ein Frühstück";
-"Tip Keka a breakfast so it can start fresh" = "Spende ein Frühstück, für einen guten Start";
+"Tip Keka a breakfast so it can start fresh" = "Spende Keka ein Frühstück, damit es gut in den Tag starten kann";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:247]: Must be 45 characters or less

File: Translations/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -233,21 +233,21 @@
...
 "Tip a lunch" = "Spende ein Mittagessen";
-"Tip Keka a lunch and feed it" = "Spende ein Mittagessen zum füttern";
+"Tip Keka a lunch and feed it" = "Spende Keka ein Mittagessen und füttere es";
 "Tip a dinner" = "Spende ein Abendessen";
-"Tip Keka a dinner so it keeps rolling" = "Spende Keka ein Abendessen und ernähre es";
+"Tip Keka a dinner so it keeps rolling" = "Spende Keka ein Abendessen, damit es weiterrollen kann";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:251]: Must be 45 characters or less

File: Translations/de.lproj/Localizable.strings [hunk 4]
```diff
@@ -233,21 +233,21 @@
...
 "Tip a bunch of food" = "Spende ein Paket Lebensmittel";
-"Tip Keka a new bunch of food to keep rolling" = "Spende ein Paket Lebensmittel und ernähre es";
+"Tip Keka a new bunch of food to keep rolling" = "Spende Keka ein neues Paket Lebensmittel zum Weiterrollen";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:253]: Must be 45 characters or less

**Review by aonez — CHANGES_REQUESTED** [review]

**Comment by agua8** [comment]
Thanks for the hint. I have adjusted the translations accordingly .

**Pull request merged**